pages/gameplan/index.js

```diff
@@ -0,0 +1,58 @@
+import GameCard from "../../components/GameCard/GameCard";
+import styled from "styled-components";
+
+const games = [
+  {
+    id: Math.random().toString(20).substring(3),
+    type: "Match",
+    name: "Max",
+    date: "22.01.10",
+    time: "19:00",
+    place: "Rothof",
+    court: "Carpet",
+  },
+  {
+    id: Math.random().toString(20).substring(3),
+    type: "Training",
+    name: "Sonja",
+    date: "22.01.15",
+    time: "20:00",
+    place: "Fidelopark",
+    court: "Sand",
+  },
+  {
+    id: Math.random().toString(20).substring(3),
+    type: "Match",
+    name: "Steffi",
+    date: "22.01.20",
+    time: "10:00",
+    place: "Sportscheck",
+    court: "Gras",
+  },
+];
+
+export default function Gameplan() {
+  return (
+    <main>
+      <StyledHeadline>Your Gameplan</StyledHeadline>
```

Comment:
This is the main heading on your page, it should be an h1. You might also consider putting an h1 heading over the main section into a header.